pkg/virt-operator/creation/components/crds.go

```diff
@@ -270,6 +271,65 @@ func NewServiceMonitorCR(namespace string, monitorNamespace string, insecureSkip
 	}
 }
 
+// NewPrometheusRuleCR returns a PrometheusRule with a group of alerts for the KubeVirt deployment.
+func NewPrometheusRuleCR(namespace string) *promv1.PrometheusRule {
+	return &promv1.PrometheusRule{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: promv1.SchemeGroupVersion.String(),
+			Kind:       "PrometheusRule",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "prometheus-kubevirt-rules",
+			Namespace: namespace,
+			Labels: map[string]string{
+				"prometheus.kubevirt.io": "",
+				"k8s-app":                "kubevirt",
+			},
+		},
+		Spec: promv1.PrometheusRuleSpec{
+			Groups: []promv1.RuleGroup{
+				{
+					Name: "kubevirt.rules",
+					Rules: []promv1.Rule{
+						{
+							Record: "num_of_running_virt_api_servers",
+							Expr:   intstr.FromString("sum(up{pod=~'virt-api-.+'})"),
+						},
+						{
+							Alert: "VirtAPIDown",
+							Expr:  intstr.FromString("num_of_running_virt_api_servers == 0"),
+							For:   "5m",
+							Annotations: map[string]string{
+								"summary": "All VirtAPI servers are down.",
+							},
+						},
+						{
+							Alert: "SingleVirtAPI",
```

Comment:
Should this be a `Record`?

Reply:
We're recording `num_of_running_virt_api_servers`... I don't think that we need to record the alert itself...
Or you meant to something else?